R/TerminatorEvals.R

```diff
@@ -40,8 +39,27 @@ TerminatorEvals = R6Class("TerminatorEvals",
     #' @return `logical(1)`.
     is_terminated = function(archive) {
       archive$n_evals >= self$param_set$values$n_evals
+    },
+
+    #' @description
```

Comment:
I thought it is not necessary to document inherited methods?

Reply:
If we overwrite them in the subclass, we have to.